LearningMissionLab/ScheduleEvent.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace LearningMissionLab
+{
+    public class ScheduleEvent
+    {
+        DateTime _dayOfTheWeek;
+        DateTime _startTimeHour;
```

Comment:
_startTimeHour should be int or uint. When you set the StartTimeHour property, you should be able to restrict the value to 23. The range is from 0 to 23.